### ISSUE TITLE
Missing Alt on Logo in footer

### ISSUE THREAD

**Issue by davidstenger** [uswds/uswds#1658]
**Issue**:
the logo is missing an alt attribute

**Solution**:
Add alt attribute to logo

**Comment by toolness** [comment]
Which footer is this?  On the [`footer--slim`](https://components.standards.usa.gov/components/detail/footer--slim.html) example, there is an `alt` attribute on an image that appears to be a logo:

```html
<div class="usa-footer-logo">
  <img class="usa-footer-slim-logo-img" src="../../dist/img/logo-img.png" alt="Logo image">
  <h3 class="usa-footer-slim-logo-heading">Name of Agency</h3>
</div>
```

However, I'm wondering if the logo has appropriate alt text--given that it's essentially decorative, as the actual name of the agency is right below it, perhaps `alt` should be set to an empty string...

**Comment by maya** [comment]
Agree with @toolness that the alt text can be set to `null` (empty) bc the logo text is already beside it. This applies to all footers. My guess as to why the alt text was there was perhaps to be overly cautious in case someone did not provide text for their logo, i.e. if the logo had text visibly.

**Comment by maya** [comment]
Closing this as there is the alt text attribute on the logo already and will fix the issue I mentioned in #2345.
